src/e_uadk.c

```diff
@@ -227,6 +219,16 @@ static int bind_fn(ENGINE *e, const char *id)
 		free(dev);
 	}
 
+	if (!uadk_bind_pkey(e))
+		fprintf(stderr, "uadk bind pkey failed\n");
+	else
+		uadk_pkey = 1;
+
+	if (!uadk_bind_ec(e))
+		fprintf(stderr, "uadk bind ec failed\n");
+	else
+		uadk_ec = 1;
+
 	return 1;
 }
 
```

Comment:
in order to support multi-platform, pls use wd_get_accel_dev,
If wd_get_accel_dev has issue in fpga, help fix it.

Reply:
I has added algorithms check supported by hisilicon in 'uadk_bind_pkey' and 'uadk_bind_ec', is it?